src/components/Main/Burger/index.js

```diff
@@ -0,0 +1,92 @@
+import styled from "styled-components";
+
+const Burger = ({ingredientsOrder,totalPrice}) => {
+    return (
+        <BurgerStyled>
+            <TotalStyled> Total price {totalPrice} UAH</TotalStyled>
+            <BurgerWrapperStyled>
+                <TopBurgerStyled
+                src={require(`../../../assets/burgerItems/top_bun.png`)}
+                />
+                {!ingredientsOrder.length && (<PropositionStyled>
+                   Start adding ingredients
+                   </PropositionStyled>)}
+                    {ingredientsOrder.map((item, index) => {
+                        return (
+                            <ItemIMGStyled 
+                            key={item + index}
```

Comment:
Try to avoid indexes as key values

https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/